datadog/resource_datadog_synthetics_test_.go

```diff
@@ -231,6 +242,34 @@ func syntheticsTestRequest() *schema.Schema {
 					Optional: true,
 					Default:  60,
 				},
+				"dnsServer": {
+					Type:     schema.TypeString,
+					Optional: true,
+				},
+			},
+		},
+	}
+}
+
+func syntheticsTestRequestClientCertificateItem() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		MaxItems: 1,
+		Required: true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"content": {
+					Type:     schema.TypeString,
+					Required: true,
+				},
+				"filename": {
+					Type:     schema.TypeString,
+					Required: true,
+				},
+				"updated_at": {
+					Type:     schema.TypeString,
+					Required: true,
```

Comment:
Are those last 2 required? Not sure why the API case about filename, or what updated_at means.

Reply:
Yes that's two fields that make sense when using the UI but not really with terraform, I'll make the `updatedAt` option automatically fill in the backend, and see what we can do for `filename`.

Reply:
I've removed the `updatedAt` option and we decided to automatically fill the `filename` (required in the backend) so it makes more sense.